src/main/java/org/broadinstitute/barclay/argparser/ExperimentalFeature.java

```diff
@@ -0,0 +1,13 @@
+package org.broadinstitute.barclay.argparser;
+
+import java.lang.annotation.*;
+
+/**
+ * Marker interface for features that are experimental and not for production use.
```

Comment:
Maybe elaborate more here?  Ex: these tools may never become stable and may be removed or changed dramatically.

Reply:
Done.